.ci/pipeline_definitions

```diff
@@ -42,6 +42,9 @@ etcd-druid:
           preprocess: 'finalize'
         release:
           nextversion: 'bump_minor'
+          git_tags:
+          - ref_template: 'refs/tags/{VERSION}'
+          - ref_template: 'refs/tags/api-{VERSION}'
```

Comment:
@AndreasBurger can you have a look if this is correct?

I want to have the standard version tag and `api-{VERSION}` tag as well.